src/screens/FeedScreen/Content.js

```diff
@@ -71,9 +71,6 @@ const Content = ({
 
       return result >= 0 ? result : 0;
     }
-    if (getCommentLength(item.latest_reactions.comment) > 0) {
-      return Math.floor(layoutHeight / lineHeight);
-    }
     return Math.round(layoutHeight / lineHeight);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the condition that checks if a comment's latest reactions length is greater than zero could potentially lead to unexpected behavior. If `item.latest_reactions.comment` is expected to be an array, and it's empty or undefined, the function will now return `Math.round(layoutHeight / lineHeight)`, which might not be the intended behavior. Please ensure this change aligns with your application's requirements.
```diff
-    return Math.round(layoutHeight / lineHeight);
+    if (getCommentLength(item.latest_reactions.comment) > 0) {
+      return Math.floor(layoutHeight / lineHeight);
+    }
+    return Math.round(layoutHeight / lineHeight);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
getCommentLength always return number, so it should not be cannot undefined